deeds-dapp-webapp/src/main/webapp/WEB-INF/metadata/about-us.html

```diff
@@ -2,6 +2,8 @@
 <meta name="description" content="#{metadata.about-us.pageDescription}">
 <link rel="canonical" href="https://www.meeds.io/about-us">
 <meta property="og:locale" content="${lang}">
+<link rel="alternate" href="https://www.meeds.io/qui-sommes-nous/" hreflang="fr"/>
```

Comment:
The proposed alternate for FR has to reference the URL without redirection. (Same for others please)

```suggestion
<link rel="alternate" href="https://www.meeds.io/qui-sommes-nous" hreflang="fr"/>
```

Reply:
done!